modules/python/tests/CMakeLists.txt

```diff
@@ -3,7 +3,11 @@ add_unit_test(LIBTEST CRITERION
   INCLUDES "${PYTHON_INCLUDE_DIR}" "${PYTHON_INCLUDE_DIRS}"
   DEPENDS syslogformat mod-python "${PYTHON_LIBRARIES}")
 
+set_property(TEST test_python_logmsg APPEND PROPERTY ENVIRONMENT "PYTHONMALLOC=malloc_debug")
+
 add_unit_test(LIBTEST CRITERION
   TARGET test_python_template
   INCLUDES "${PYTHON_INCLUDE_DIR}" "${PYTHON_INCLUDE_DIRS}"
   DEPENDS mod-python "${PYTHON_LIBRARIES}" syslogformat)
+
+set_property(TEST test_python_template APPEND PROPERTY ENVIRONMENT "PYTHONMALLOC=malloc_debug")
```

Comment:
can you also add this to Makefile.am?

just assign PYTHONMALLOC to the same value and then "export PYTHONMALLOC" in Makefile.am

Reply:
Like this https://github.com/balabit/syslog-ng/pull/2866/commits/31d7c93da0653ac41330d4649268cc3c52906831#diff-68c196e53bce5b3146d2b805debf05dcR1 ?